src/osd/PrimaryLogPG.cc

```diff
@@ -2517,6 +2517,30 @@ void PrimaryLogPG::do_op(OpRequestRef& op)
   maybe_force_recovery();
 }
 
+bool PrimaryLogPG::need_skip_handle_manifest(const OpRequestRef& op_req,
+                                             bool is_chunked)
```

Comment:
Probably, this is_chunked can be removed from the input arguments.

Reply:
When initially targeted filtering cls cas op, is_chunked is for distinguishing the type of manifest.
However, I don't think it is necessary to filter all operations in general.
I'll fix it.

Reply:
I think adding the following lines below `case object_manifest_t::TYPE_CHUNKED:` is enough.

    if (op_req->may_read() && !op_req->may_read_data() && !op_req->may_write()) {
       return true;
     }

Reply:
Oh I see what you mean it. Thank you for letting me know.
Then it seems better to filter in the switch-case than to call need_skip_handle_manifest() like what you said.